### PR TITLE
Add missing template variables

### LINKED ISSUE #125: Template attributes missing in the docs
Old deprecated documentation https://github.com/sulu-io/docs/blob/master/developer-documentation/300-webspaces/templates.md

Maybe part of Creating a Twig Template?

### PR DESCRIPTION
**Pull request by alexander-schranz**
| Q | A
| --- | ---
| Fixed tickets | fixes #125
| License | MIT

#### What's in this PR?

Add missing template variables

#### Why?

The are only in the old documentation available.

### CODE REVIEW

File: book/twig.rst [hunk 1]
```diff
@@ -92,6 +92,31 @@ The excerpt is available in:
 
     {{ extension.excerpt }}
 
+View
+^^^^
+
+In the view var Sulu writes the view data of the defined properties 
+in your template. Example the `media_selection` store the displayOption there.
```
Comment (wachterjohannes) [inline book/twig.rst]: `As an example the ...`

**Comment by wachterjohannes** [comment]
technically its good

**Comment by alexander-schranz** [comment]
@wachterjohannes PR updated

File: book/twig.rst [hunk 2]
```diff
@@ -92,7 +92,7 @@ You could include the SEO meta tags like this:
         defaultLocale: request.defaultLocale
     } -%}
 
-The excerpt is available in:
+The excerpt data is available with:
```
Comment (danrot) [inline book/twig.rst]: `from` would be the better option IMO.

File: book/twig.rst [hunk 3]
```diff
@@ -102,6 +102,34 @@ The excerpt is available in:
     {{ extension.excerpt.icon[0].thumbnails['50x50'] }}
     {{ extension.excerpt.images[0].thumbnails['300x300'] }}
 
+View
+^^^^
+
+In the view var Sulu writes the view data of the defined properties
```
Comment (danrot) [inline book/twig.rst]: mark view as code with ``` and don't abbreviate `variable`.

File: book/twig.rst [hunk 4]
```diff
@@ -102,6 +102,34 @@ The excerpt is available in:
     {{ extension.excerpt.icon[0].thumbnails['50x50'] }}
     {{ extension.excerpt.images[0].thumbnails['300x300'] }}
 
+View
+^^^^
+
+In the view var Sulu writes the view data of the defined properties 
+in your template. As an example the `media_selection` store the
```
Comment (danrot) [inline book/twig.rst]: `stores`

File: book/twig.rst [hunk 5]
```diff
@@ -102,6 +102,34 @@ The excerpt is available in:
     {{ extension.excerpt.icon[0].thumbnails['50x50'] }}
     {{ extension.excerpt.images[0].thumbnails['300x300'] }}
 
+View
+^^^^
+
+In the view var Sulu writes the view data of the defined properties 
+in your template. As an example the `media_selection` store the 
+displayOption there.
```
Comment (danrot) [inline book/twig.rst]: `displayOption`

**Comment by alexander-schranz** [comment]
@danrot corrections done.

File: book/twig.rst [hunk 6]
```diff
@@ -102,6 +102,34 @@ The excerpt is available in:
     {{ extension.excerpt.icon[0].thumbnails['50x50'] }}
     {{ extension.excerpt.images[0].thumbnails['300x300'] }}
 
+View
+^^^^
+
+In the ``view`` variable Sulu writes the view data of the defined properties 
+in your template. As an example the ``media_selection?`` stores the
```
Comment (danrot) [inline book/twig.rst]: No question mark.

**Pull request merged**